### DISCUSSION TITLE
Hinode and mod mgallery

### DISCUSSION THREAD

**Discussion by maciejglowackimg** [gethinode/hinode#764] [Help]
@markdumay  One more favour. I would like to add mgallery to "hinode". What is the correct procedure for adding a module? Should I change the approach to configuration files now I have a "hugo.toml" file or should I change it to "config/_default/hugo.toml", "config/_default/params/params.toml"? Should I use "hugo mod init" to install?

Could you explain this in a few sentences? Thank you very much.

**Comment by markdumay** [comment]
It depends if `mgallery` is a compatible module (it is not maintained by me). To configure compatible modules, you should check the [module configuration ](https://gethinode.com/docs/configuration/modules/)in the docs. Or else, check the [module development](https://gethinode.com/docs/advanced-settings/module-development/) documentation on how to develop a Hugo module that is compatible with Hinode.
